libraries/SITL/SIM_Aircraft.cpp

```diff
@@ -458,6 +459,29 @@ void Aircraft::fill_fdm(struct sitl_fdm &fdm)
         set_speedup(sitl->speedup);
         last_speedup = sitl->speedup;
     }
+
+    // for EKF comparison log relhome pos and velocity at loop rate
+    static uint32_t last_sim2_ms;
+    uint32_t now_ms = AP_HAL::millis();
+    if (now_ms - last_sim2_ms >= AP::scheduler().get_loop_period_s()*1000) {
```

Comment:
Why not just look at `AP::scheduler().ticks()` instead?

Reply:
good idea, done